### ISSUE TITLE
Missing offenses in `Style/TrailingMethodEndStatement`

### ISSUE THREAD

**Issue by garettarrowood** [bbatsov/rubocop#5343]
Offenses are missed by `Style/TrailingMethodEndStatement`.

--------

## Expected behavior

This example to raise an offense:

```ruby
class Foo
  def some_method
  foo; end
end
```

## Actual behavior

Snippet does not raise offense. Thus, it is also not autocorrected.

## Steps to reproduce the problem

Use default settings and run `rubocop` on snippet above.

## RuboCop version

```
0.52.1 (using Parser 2.4.0.2, running on ruby 2.3.3 x86_64-darwin15)
```
